addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4220,7 +4220,16 @@ export class OdooEditor extends EventTarget {
                 this.execCommand('insert', fragment);
             }
         } else if (files.length && targetSupportsHtmlContent) {
-            this.addImagesFiles(files).then(html => this.execCommand('insert', this._prepareClipboardData(html)));
+            this.addImagesFiles(files).then(html => {
+                const imageNodes = this.execCommand('insert', this._prepareClipboardData(html));
```

Comment:
Are we sure this will return the node(s) all the time? (otherwise it will crash later)
I guess it's the case here since the previous check `targetSupportsHtmlContent` ? (And other case thanks to `isHtmlContentSupported` ?)

Reply:
I added the condition on both tests, just in case.